### PR TITLE
DCOS-15921 (4 of 4): Implement code example for design system checkbox c…

### PR DESCRIPTION
**Pull request by tnguyen0**
This PR uses the code example component for checkboxes in the design system.

This PR depends on: #2312, #2313, #2314 

See branch `tommy/design-system/component-example-ace-editor` for the complete implementation.
In the UI: `Design System` --> `Components` --> `Checkboxes` --> `Code Tab`

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: plugins/design-system/styles/styles.less [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import '../../../node_modules/codemirror/lib/codemirror.css';
+@import 'components/component-example/styles.less';
```
Comment (bstavroulakis) [inline plugins/design-system/styles/styles.less]: I get an error Error: 'components/component-example/styles.less' wasn't found.

Comment (tnguyen0) [inline plugins/design-system/styles/styles.less]: @bstavroulakis The PR you are looking at depends on a few other PRs, so it won't build until those other PRs are merged. The working branch is `tommy/component-example`. It is rather large, so I broke it up into multiple PRs.

Comment (bstavroulakis) [inline plugins/design-system/styles/styles.less]: @tnguyen0 I see, sorry I'll recheck!

**Comment by tnguyen0** [comment]
Closing this and will reopen a new one with the changes discussed in today's meeting.

**Pull request closed without merging**